cmd/dhcp-sync/dhcp-sync.go

```diff
@@ -184,10 +184,10 @@ var Cmd = &cobra.Command{
 					if !ok {
 						return
 					}
-					klog.Info("event:", event)
+					klog.V(2).Info("event:", event)
 					if event.Op&fsnotify.Write == fsnotify.Write {
-						klog.Info("modified file:", event.Name)
-						klog.Info("restarting the dhcpd service")
+						klog.V(2).Info("modified file:", event.Name)
```

Comment:
Suggested usage of `Infof` over `Info` although the latter achieves the same objective. 
This can be done to keep changes in-line with the rest of the project.

Reply:
Done.